backend/src/ee/services/secret-approval-request/secret-approval-request-service.ts

```diff
@@ -447,8 +447,8 @@ export const secretApprovalRequestServiceFactory = ({
     );
     const hasMinApproval =
       secretApprovalRequest.policy.approvals <=
-      secretApprovalRequest.policy.approvers.filter(
-        ({ userId: approverId }) => reviewers[approverId.toString()] === ApprovalStatus.APPROVED
+      secretApprovalRequest.policy.approvers.filter(({ userId: approverId }) =>
+        approverId ? reviewers[approverId.toString()] === ApprovalStatus.APPROVED : null
```

Comment:
Why are we doing approverId.toString() when we are already checking if it's defined, by using the ternary operator?